dynamiq/connections/connections.py

```diff
@@ -855,6 +856,61 @@ def connect(self):
         pass
 
 
+class MilvusDeploymentType(str, enum.Enum):
+    """
+    Defines various deployment types for different Milvus deployments.
+
+    Attributes:
+        LOCAL_FILE (str): Represents a local file-based deployment.
+        DOCKER (str): Represents a Docker-based deployment on localhost.
+        ZILLIZ_CLOUD (str): Represents a cloud deployment on Zilliz Cloud.
+    """
+
+    LOCAL_FILE = "local_file"
```

Comment:
Those types are so specific and don't cover other possible deployments variations.
So let's use more general like `FILE` and `HOST`:
- `FILE` - local file with validation for `.db` suffix
- `HOST` could be any cloud, cluster, or single machine deployment with or without token. So all validation could be done by `MilvusClient`. In `HOST` case client connect would looks like:
 ```python
 milvus_client = MilvusClient(uri=self.uri, token=self.api_key)
 ```